lib/src/testCommon/kotlin/org/dif/fixtures/BenchCommon.kt

```diff
@@ -0,0 +1,155 @@
+package org.dif.fixtures
+
+import org.dif.DIDComm
+import org.dif.mock.AliceSecretResolverMock
+import org.dif.mock.BobSecretResolverMock
+import org.dif.mock.DIDDocResolverMock
+import org.dif.model.PackEncryptedParams
+import org.dif.model.PackSignedParams
+import org.dif.model.UnpackParams
+import kotlin.system.measureTimeMillis
+
+class BenchRes(
+    val operations: Int,
+    val timeInMs: Long,
+    val opName: String = "noname",
+) {
+    val avrg = timeInMs.toFloat() / operations
+    val thrpt = 1 / avrg
+
+    fun toStr(): String {
+        return String.format(
+            "benchmark of %-45s took %7s ms, %7s ops, %10s ops/ms, %7s mss/op",
+            "'${this.opName}'", this.timeInMs, operations, this.thrpt, this.avrg
+        )
+    }
+}
+
+class BenchCommon {
+    companion object {
+        val didComm_def: DIDComm = DIDComm(DIDDocResolverMock(), AliceSecretResolverMock())
+
+        val packSignedRes = didComm_def.packSigned(
```

Comment:
Camel case is usually used for Java/Kotlin vars and methods

Reply:
fixed